IrvinRamiA/Integrator-Project
# Power Module Integration

#4 

Definition of done:

- [x] Modify the PWM module to control the duty cycle with a potentiometer through the ADC module.

QA:
- [x] Behaves as expected. 

Connect PWM output to power board with the next configurations:
Pin Vcc - 12V
Pin 3.3V - 3.3V
Pin PWM - P46 from Renesas Synergy board
Pin M_M - 12V
Pin GND - GND

Verify visually that fan speed increases when duty cycle from PWM increases and when fan speed decreases, duty cycle dicreases.
